portal-services/src/main/resources/portal-services-openapi.yaml

```diff
@@ -10403,6 +10403,31 @@ paths:
                 $ref: '#/components/schemas/PortalUser'
         500:
           $ref: '#/components/responses/GenericApiError'
+          
+  /portal/portalUser/customerIdsForUsername:
```

Comment:
this needs to match the annotation in PortalUserPortalController - the path should end with "/portalUser/usersForUsername"